### DISCUSSION TITLE
[Question] TV Channel Url m3u8 cannot be played on flet Video

### DISCUSSION THREAD

**Discussion by wanfir** [flet-dev/flet#3965] [Q&A] [answered]
### Question

Hi, I am in confusion. I tried to play some tv url on flet video player but some of the tv channel urls cannot be played. For example channel 1 can be played on the using flet Video class but channel 2 cannot be played.
However, when I play channel 2 url in vlc media player, it can play without any problem.

note: channel 1: https://d25tgymtnqzu8s.cloudfront.net/smil:tv1/playlist.m3u8?id=1
channel 2: https://ythls.armelin.one/channel/UC5dYmq91e5_g54krpO06NJw.m3u8

Why is this happen? Both channel are m3u8 playlist.

### Code sample

```python
class VideoPlayerPlay(ft.Container):
    def __init__(self, url):
        super().__init__()
        self.content = ft.Video(
            expand=True,
            playlist=ft.VideoMedia(url),
            playlist_mode=ft.PlaylistMode.LOOP,
            fill_color=ft.colors.BLUE_400,
            aspect_ratio=16/9,
            autoplay=True,
            filter_quality=ft.FilterQuality.HIGH,
            muted=False,
            on_loaded=lambda e: print("Video loaded successfully!"),
            on_error=lambda e: print("Error Occured Please Try Again"),
            on_enter_fullscreen=lambda e: print("Video entered fullscreen!"),
            on_exit_fullscreen=lambda e: print("Video exited fullscreen!"),
            visible=True)
```

### ------------------------------------------------------

- [ ] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/bdb01319-f928-4b24-8f3f-827a3ccac81c)
It works on macOS, but it takes likes 5-10 seconds to load. Reducing the filter quality might help reduce this delay a little.

How do you run the app on your TV?

**Reply by wanfir** [reply]
Thank you for the advise. I put filter quality as medium and all the channel are play smoothly.

I run it on windows.
